ccmlib/scylla_node.py

```diff
@@ -714,6 +714,10 @@ def import_bin_files(self):
                 self._launch_env = dict(os.environ)
                 self._launch_env['LD_LIBRARY_PATH'] = dbuild_so_dir
 
+                if scylla_mode == 'debug':
+                    cmd = ['strip', dst]
```

Comment:
Better to check the file size instead of the mode.

Reply:
right, will fix.